pennylane_lightning/src/simulator/Measures.hpp

```diff
@@ -162,6 +162,7 @@ class Measures {
     /**
```

Comment:
Could you please update the following lines,
- line 45: `@tparam SVType Statevector type to register`,
- line 197 ` @tparam op_type Operation type.`
- line 280 missing `op_type`